ServerCore/Areas/Identity/UserAuthorizationPolicy/IsAuthorOfPuzzle.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using ServerCore.DataModel;
+
+namespace ServerCore.Areas.Identity.UserAuthorizationPolicy
+{
+    public class IsAuthorOfPuzzleRequirement : IAuthorizationRequirement
```

Comment:
Please add an XML comment on what this and the other requirements mean?

Reply:
Done